charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -30,4 +30,77 @@ spec:
   ingress:
     - {}
   {{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-prometheus
+  namespace: kyverno
+  labels:
+    app: kyverno
+spec:
+  podSelector:
+    matchLabels:
+      app: kyverno
+  ingress:
+  - from:
+    {{- with .Values.networkPolicy.prometheusNamespaceLabels }}
+    - namespaceSelector:
+        matchLabels:
+            {{ toYaml . | nindent 10 }}
+    {{- end }}
+    {{- with .Values.networkPolicy.prometheusPodLabels }}
+    - podSelector:
+        matchLabels:
+            {{ toYaml . | nindent 10 }}
+    {{- end }}
+    ports:
+    # Webhook access
+    - protocol: TCP
+      port: 9443
+    # Allow prometheus scrape for metrics
+    - protocol: TCP
+      port: 8000
+  policyTypes:
+    - Ingress
+    - Egress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-apiserver
```

Comment:
Missing `namespace` here, and should have value be `{{ include "kyverno.namespace" . }}`